src/snowflake/snowpark/modin/plugin/docstrings/io.py

```diff
@@ -343,6 +343,125 @@ def read_xml():
 def json_normalize():
     """
     Normalize semi-structured JSON data into a flat table.
+
+    Parameters
+    ----------
+    data : dict or list of dicts
+        Unserialized JSON objects.
+    record_path : str or list of str, default None
+        Path in each object to list of records. If not passed, data will be assumed to be an array of records.
+    meta : list of paths (str or list of str), default None
+        Fields to use as metadata for each record in resulting table.
+    meta_prefix : str, default None
+        If True, prefix records with dotted (?) path, e.g. foo.bar.field if meta is [‘foo’, ‘bar’].
+    record_prefix : str, default None
+        If True, prefix records with dotted (?) path, e.g. foo.bar.field if path to records is [‘foo’, ‘bar’].
+    errors : {‘raise’, ‘ignore’}, default ‘raise’
+        Configures error handling.
+        - ‘ignore’ : will ignore KeyError if keys listed in meta are not always present.
+        - ‘raise’ : will raise KeyError if keys listed in meta are not always present.
+    sep : str, default ‘.’
+        Nested records will generate names separated by sep. e.g., for sep=’.’, {‘foo’: {‘bar’: 0}} -> foo.bar.
+    max_level : int, default None
+        Max number of levels(depth of dict) to normalize. if None, normalizes all levels.
+
+    Returns
+    -------
+    frame : DataFrame
```

Comment:
```suggestion
    A pandas on Snowflake DataFrame.
```

Reply:
we mostly use either "DataFrame" or "Snowpark pandas DataFrame" in other docstrings, so we should pick one of those for consistency

Reply:
Agree with Jonathan. We generally don't use "pandas on Snowflake" in the codebase.